packages/website/src/css/custom.css

```diff
@@ -60,17 +61,7 @@ html[data-theme='dark']:root {
   scrollbar-color: #454a4d #202324;
 
   --docsearch-muted-color: #aaa;
-}
-
-.docusaurus-highlight-code-line {
```

Comment:
This is a breaking change: the class name has been removed in Docusaurus. We are using CSS vars instead.